R/app_server.R

```diff
@@ -8,6 +8,7 @@
   questions_df <- .get_questions()
 
   output$questions <- .render_questions(questions_df)
+  session$onSessionEnded(stopApp)
```

Comment:
```suggestion
  session$onSessionEnded(shiny::stopApp)
```

Reply:
That needs to be namespaced, right?

Reply:
When I execute run_app, I get the message "Loading required package: shiny".
Shiny seems to be loaded automatically with any shiny app.

Reply:
Technically it's loaded, but it... doesn't feel right for a package function to call an un-namespaced function. For example, as I refine testing, shiny might NOT be loaded in a test context.

Reply:
But I do not quite understand why. There is no `#' @import ` anywhere. It might be a `golem` thing.